troi/patches/daily_jams.py

```diff
@@ -62,8 +61,11 @@ def create(self, inputs):
         raw_recs_lookup = troi.musicbrainz.recording_lookup.RecordingLookupElement()
         raw_recs_lookup.set_sources(raw_recs)
 
+        recent_listens_lookup = troi.listenbrainz.listens.RecentListensTimestampLookup(user_name, days=2)
+        recent_listens_lookup.set_sources(raw_recs_lookup)
+
         latest_filter = troi.filters.LatestListenedAtFilterElement(14)
```

Comment:
Make this a constant please, since this is a critical tuning value for the patch.